tasks/closePosition/index.ts

```diff
@@ -144,17 +153,19 @@ task('closePosition', 'Close stETH position on AAVE')
       },
     )
 
-    const strategyReturn = await strategies.aave.closeStEth(
+    const positionMutation = await strategies.aave.close(
```

Comment:
Some tests is called `positionMutation` and in some `closePositionMutation`. Let's stick with one approach. Again IMO it should be just `closePosition`.